docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md

```diff
@@ -41,10 +41,31 @@ Following are the [fields](/docs/manage/fields/) which will be created as part o
 
 In this step, you will configure the yaml required for macOS Collection.
 
-1. (Optional) You can add a custom tag, if desired. (There are no parameters required for the macOS app to install.)
-2. Click on the **Download YAML File** button to get the yaml file.
+(Optional) You can add a custom tag, if desired. (There are no parameters required for the macOS app to install.)
 
-<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-YAML.png' alt="YAML" />
+#### Enable process metric collection (Optional)
```

Comment:
Copy over any changes here that were made in the Linux doc

Reply:
I think you are referring to [this](https://github.com/SumoLogic/sumologic-documentation/pull/3304/files#diff-55d6813f44f119335535210cfe052874036cdba7aa2b1581edbb04eafdf9dfceR68) section in linux doc. But this is applicable only to linux collection. And does not apply for windows and mac

Reply:
@sumoanema Yes correct that's the section. To confirm, do you mean @JV0812 needs to delete this section from docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md and docs/integrations/hosts-operating-systems/opentelemetry/windows-opentelemetry.md?